### DISCUSSION TITLE
Error SyntaxError: future feature annotations is not defined

### DISCUSSION THREAD

**Discussion by albertocabasvidani** [tryptech/obs-zoom-and-follow#102] [Q&A]
Hello, 
I installed Python 3.6. OBS is loading it correctly.
I installed pywin and it confirmed everything worked. 
I added the script to OBS and I get this error message:
![image](https://user-images.githubusercontent.com/129092135/227996959-9b5edf83-4f04-40f4-880c-5d583b673e64.png)

What should I do?

**Comment by kmanan** [comment]
Getting the same error. Were you able to resolve it?

**Comment by Kalmat** [comment]
Hi both! I think that either installing PyWinCtl v0.0.38, or upgrading to Python 3.10, should fix the issue
